src/App.js

```diff
@@ -1,24 +1,26 @@
 import React from "react";
```

Comment:
Please change the toolbar's background to something that contrasts better with the buttons:
```css
  background: #abcdef;

```